src/ontology/config/external-content-robot-headers.json

```diff
@@ -0,0 +1,11 @@
+{
+    "mondo_id": "ID",
+    "report_ref": "A oboInOwl:hasDbXref",
+    "report_ref_source": ">A oboInOwl:source",	
+    "preferred_name": "A oboInOwl:hasExactSynonym",
+    "preferred_name_source": ">A oboInOwl:hasDbXref",	
+    "synonym_type": ">A oboInOwl:hasSynonymType",	
+    "subset": "AI oboInOwl:inSubset",	
+    "subset_source": ">A oboInOwl:source",	
+    "subset_source2": ">A oboInOwl:source"
+}
```

Comment:
Can this have 3 `">A oboInOwl:source"`? I gather yes, but not comprehending this at the moment.

Reply:
Yes, these would be three individual source annotations. If you look at an example class in Mondo, you can often find more than one source on each term.

Reply:
yeah, I've seen that. given the issue with "double comments" when obsoleting a term, i didn't know ROBOT could handle multiple sources.